### PR TITLE
Fix for deploy of undefined resource

### PR DESCRIPTION
**Pull request by bartv**

### CODE REVIEW

**Comment by wouterdb** [comment]
also document state machine

File: src/inmanta/export.py [hunk 1]
```diff
@@ -440,8 +440,7 @@ def call():
                 LOGGER.debug("Uploaded file with hash %s" % hash_id)
 
         # Collecting version information
-        version_info = {"export_metadata": metadata,
-                        "model": model}
+        version_info = {"export_metadata": metadata}
```
Comment (bartv) [inline src/inmanta/export.py]: This does not belong in this patch
